docs/uefi.md

```diff
@@ -0,0 +1,42 @@
+# UEFI Boot
+
+Cloud Hypervisor supports UEFI boot through the utilization of the EDK II based UEFI firmware. 
+
+## Building UEFI Firmware
+
+To avoid any unnecessary issues, it is recommended to use Ubuntu 18.04 and its default toolset. Any other compatible Linux distribution is otherwise suitable, however it suggests itself to use a temporary docker container with Ubuntu 18.04 for a quick worry-free build on some existing Linux machine.
+
+The commands below afford to compile the OVMF firmware suitable for the Cloud Hypervisor.
+
+```shell
+sudo apt-get update
+sudo apt-get install uuid-dev nasm iasl build-essential python3-distutils git
+
+git clone https://github.com/cloud-hypervisor/edk2 -b ch
+cd edk2
+. edksetup.sh
+git submodule update --init
+
+echo "ACTIVE_PLATFORM=OvmfPkg/OvmfCh.dsc" >> Conf/target.txt
+echo "TARGET_ARCH=X64" >> Conf/target.txt
+echo "TOOL_CHAIN_TAG=GCC5" >> Conf/target.txt
+
+make -C ./BaseTools
+build
+```
+
+After the successful build, the resulting firmware binaries are available under `Build/OvmfCh/DEBUG_GCC5/FV` underneath the edk2 checkout.
+
+## Using OVMF Binaries
+
+Any UEFI capable image can be booted using the Cloud Hypervisor specific firmware. With regard to Windows, only UEFI boot is supported and therefore the OVMF is mandatory there.
```

Comment:
`Any UEFI capable image can be booted using the Cloud Hypervisor specific firmware. Windows guests only support UEFI boot, therefore OVMF is mandatory there.`

Reply:
This way it might sound, that Windows only supports UEFI boot. Just added "under Cloud Hypervisor" to the fi for clarity. Thanks